so_pysm_models/tests/test_extragalactic.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import healpy as hp
+
+from .. import utils
+from .. import WebSkyCIB, WebSkySZ
+
+
+def test_cib(tmp_path, monkeypatch):
+
+    monkeypatch.setattr(utils, "PREDEFINED_DATA_FOLDERS", [str(tmp_path)])
+    nside = 4
+    shape = hp.nside2npix(nside)
+
+    path = tmp_path / "websky" / "0.3"
+    path.mkdir(parents=True)
+    hp.write_map(path / "cib_0094.fits", np.zeros(shape, dtype=np.float32))
+    hp.write_map(path / "cib_0100.fits", np.ones(shape, dtype=np.float32))
+
+    interp = WebSkyCIB("0.3", "uK_RJ", nside, interpolation_kind="linear")
+
+    interpolated_map = interp.signal(nu=97)
+    np.testing.assert_allclose(
+        np.interp(97, [94, 100], [0, 1]) * np.ones((1, shape)), interpolated_map
+    )
+
+def test_ksz(tmp_path, monkeypatch):
+
+    monkeypatch.setattr(utils, "PREDEFINED_DATA_FOLDERS", [str(tmp_path)])
+    nside = 4
+    shape = hp.nside2npix(nside)
+
+    path = tmp_path / "websky" / "0.3"
+    path.mkdir(parents=True)
+    hp.write_map(path / "ksz.fits", np.ones(shape, dtype=np.float32))
+    hp.write_map(path / "cib_0100.fits", np.ones(shape, dtype=np.float32))
+
+    ksz = WebSkySZ("0.3", sz_type="kinetic", target_nside=nside)
+
+    ksz_map = ksz.signal(nu=100)
+    np.testing.assert_allclose(
+        np.ones(ksz_map.shape)*0.777228, ksz_map, rtol=1e-4
+    )
+
```

Comment:
@marcelo-alvarez I have added 2 unit tests for `ksz` and `tsz`.
I do not want to actually download the real maps, so I create nside=4 maps full of ones and I trick  the classes into loading those.
Then I executed the code once, got the result, for example 0.777228 in the `ksz` case and make sure that future changes do not mess this up.
Is this ok? do you have in mind a better way to test this?

Reply:
for now this sounds good to me, I will let you know if I think of a better way.